integration-tests/maven/src/test/java/io/quarkus/maven/it/NativeImageIT.java

```diff
@@ -16,13 +16,13 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Assumptions;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
 
 import io.quarkus.maven.it.verifier.MavenProcessInvocationResult;
 import io.quarkus.maven.it.verifier.RunningInvoker;
 
-/**
- *
- */
+// ensure that this test only runs when GraalVM is configured
+@EnabledIfEnvironmentVariable(named = "GRAALVM_HOME", matches = ".*")
```

Comment:
So we don't run it on CI with Docker? I don't think GRAALVM_HOME is set in this case?

Reply:
We do, I checked the docker image that we use and it does set it

Reply:
The Docker image sets it. But is it set when the test is executed?

Reply:
It's part of the Docker image, so it's present when the container is run

Reply:
I might be missing something but my understanding is the test is not executed inside the Docker image and thus the `GRAALVM_HOME` is not set. Am I missing something?

Reply:
Aha, I see what you are saying! Indeed you are absolutely right! I'll try and think of some other way to do this.
However I don't knoq how this test ever passed in the first place... The reason I disabled it is because the PR was failing on it, because of course native-image doesn't exist

Reply:
I pushed a change. WDYT?

Reply:
Fixed a different way